src/Transports.Subscriptions.Abstractions/SubscriptionManager.cs

```diff
@@ -93,11 +86,15 @@ public Task UnsubscribeAsync(string id)
             ExecutionResult result;
             using (var scope = _serviceScopeFactory.CreateScope())
             {
-                result = await _executer.ExecuteAsync(
-                    payload,
-                    context,
-                    scope.ServiceProvider
-                ).ConfigureAwait(false);
+                result = await _executer.ExecuteAsync(new ExecutionOptions
+                {
+                    Query = payload.Query,
+                    OperationName = payload.OperationName,
+                    Variables = payload.Variables,
+                    Extensions = payload.Extensions,
+                    RequestServices = scope.ServiceProvider,
+                    UserContext = context,
```

Comment:
Add cancellation token?

Reply:
Well, my code certainly does.  I don't see an obvious way to add it here.